tests/block/035

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-2.0
+# Copyright 2024 Google LLC
+#
+# Test fairness if hardware queues share a tag set across request queues.
+
+. tests/block/rc
+. common/null_blk
+
+DESCRIPTION="shared tag set fairness"
+TIMED=1
+
+requires() {
+	_have_fio
+	_have_null_blk
+}
+
+test() {
+	local runtime=30
+
+	echo "Running ${TEST_NAME}"
+
+	local nullb_params=(
+		blocking=1
+		irqmode=2
+		nr_devices=0
+		shared_tags=1
+	)
+	if ! _init_null_blk "${nullb_params[@]}"; then
+		echo "Loading null_blk failed"
+		return 1
+	fi
+	local nullb_params=(
+		completion_nsec=$((10**6)) # 1 ms
+		hw_queue_depth=64
+		memory_backed=1
+		size=1                     # MiB
+		submit_queues=1
+		power=1
+	)
+	if ! _configure_null_blk nullb0 "${nullb_params[@]}"; then
+		echo "Configuring null_blk failed (1/2)"
+		return 1
+	fi
+	local nullb_params=(
+		completion_nsec=$((10**8)) # 100 ms
+		hw_queue_depth=64
+		memory_backed=1
+		size=1                     # MiB
+		submit_queues=1
+		power=1
+	)
+	if ! _configure_null_blk nullb1 "${nullb_params[@]}"; then
+		echo "Configuring null_blk failed (2/2)"
+		return 1
+	fi
+	local fio_output=${RESULTS_DIR}/block/fio-output-block-035.txt
+	fio --rw=randwrite --ioengine=io_uring --iodepth=64 \
+		--direct=1 --runtime=${runtime} --time_based=1 \
+		--name=nullb0 --filename=/dev/nullb0 \
+		--name=nullb1 --filename=/dev/nullb1 \
+		--output="${fio_output}" \
+		>>"$FULL"
```

Comment:
Nit: Stdout and stderr of fio are separated to two files, ${fio_output} and $FULL. IMO, this looks too much and single file $FULL.

Reply:
`>>"$FULL"` has been changed into `>>"$FULL" 2>&1`